### PR TITLE
Implementing Review Changes

### PR DESCRIPTION
**Pull request by elink21**
**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: .github/gh-actions-self-hosted-runners/helper-functions/cloud-functions/monitorRunnersStatus/index.js [hunk 1]
```diff
@@ -37,14 +37,28 @@ async function monitorRunnerStatus() {
...
         //Filtering BEAM runners
-        let beamRunners = runners.data.runners.filter(runner => {
-            return runner.labels.find(label => label.name == "beam")
+        let beamRunners = runners.filter(runner => {
+            for (let label of runner.labels) {
+                if (label.name == "beam") {
+                    return true;
+                }
+            }
+            return false;
```
Comment (dannymartinm) [inline .github/gh-actions-self-hosted-runners/helper-functions/cloud-functions/monitorRunnersStatus/index.js:61]:
```suggestion
            return runner.labels.find(label => label.name == "beam")
```

File: .github/gh-actions-self-hosted-runners/helper-functions/cloud-functions/removeOfflineRunners/index.js [hunk 2]
```diff
@@ -37,14 +37,24 @@ async function removeOfflineRunners() {
...
+            pageCounter++
+        } while(pageRunners.data.runners.length!=0)
+        
 
         //Filtering BEAM runners
-        let beamRunners = runners.data.runners.filter(runner => {
+        let beamRunners = runners.filter(runner => {
             for (let label of runner.labels) {
-                if (label.name == "Linux") {
+                if (label.name == "beam") {
```
Comment (dannymartinm) [inline .github/gh-actions-self-hosted-runners/helper-functions/cloud-functions/removeOfflineRunners/index.js:57]:
```suggestion
              return runner.labels.find(label => label.name == "beam")
```
I'd suggest replacing the `for` and `i`f with this function

**Review by dannymartinm — CHANGES_REQUESTED** [review]

File: .github/gh-actions-self-hosted-runners/helper-functions/cloud-functions/monitorRunnersStatus/index.js [hunk 3]
```diff
@@ -70,6 +88,7 @@ async function monitorRunnerStatus() {
 }
 
 functions.http('monitorRunnerStatus', (req, res) => {
+    validateEnvSet(["APP_ID","PEM_KEY","CLIENT_ID","CLIENT_SECRET","APP_INSTALLATION_ID","ORG"])
```
Comment (dannymartinm) [inline .github/gh-actions-self-hosted-runners/helper-functions/cloud-functions/monitorRunnersStatus/index.js:91]: I'd suggest to set the ENV_VARS as a constant at the top of the script. Same case for the rest of the files. THank you!

**Review by dannymartinm — CHANGES_REQUESTED** [review]

File: .github/gh-actions-self-hosted-runners/helper-functions/cloud-functions/monitorRunnersStatus/index.js (hunk 3 above)
Comment (elink21) [reply .github/gh-actions-self-hosted-runners/helper-functions/cloud-functions/monitorRunnersStatus/index.js:91]: Oki, will implement the changes, thanks!

**Review by dannymartinm — APPROVED** [review]
LGTM, thank you!

**Pull request merged**